test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxServiceTests.cs

```diff
@@ -839,6 +839,32 @@ public async Task ValidateStreamingJob()
             dacfxRequestContext.VerifyAll();
         }
 
+        /// <summary>
+        /// Verify Object Types Dictionary items with ObjectType Enum members
+        /// </summary>
+        /// <returns></returns>
+        [Test]
+        public void ValidateObjectTypesOptionswithEnum()
+        {
+            DeploymentOptions options = new DeploymentOptions();
+
+            // Verify the object types dictionaty should exists
+            Assert.That(options.ObjectTypesDictionary, Is.Not.Null, "Object types dictionary is empty");
+
+            // Verify that the objects dictionary has all the item from Enum
+            Assert.That(options.ObjectTypesDictionary.Count, Is.EqualTo(Enum.GetNames(typeof(ObjectType)).Length), "Object types dictionary do not have all the items from ObjectType enum");
```

Comment:
Can you update the error message to say which types are missing?  You may be able to use https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.except?view=net-6.0

Reply:
Good one! updated and tested that i can see the missing objects in the message!